net-misc/sipp/sipp-3.7.1.ebuild

```
@@ -0,0 +1,45 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
```

Always use the latest available EAPI.
```suggestion
EAPI=8
```